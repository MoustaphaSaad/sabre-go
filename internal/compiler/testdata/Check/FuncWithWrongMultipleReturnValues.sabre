package main

func foo() (int) {
    return 1, 1.5
}
