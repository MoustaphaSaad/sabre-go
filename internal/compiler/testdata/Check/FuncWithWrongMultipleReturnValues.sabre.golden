>> 	    return 1, 1.5
>> 	    ^^^^^^^^^^^^^ 
Error[internal/compiler/testdata/Check/FuncWithWrongMultipleReturnValues.sabre:4:5]: expected '1' return values, but found '2'
